modules/migration-converting-the-storage-class-of-persistent-volumes-cam.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-converting-the-storage-class-of-persistent-volumes-cam_{context}"]
+= Converting the storage class of persistent volumes in the {mtc-short} web console
+
+You can convert the storage class of persistent volumes (PVs) by migrating them within the same cluster. To do so, you must create and run a migration plan in the {mtc-full} ({mtc-short}) web console.
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on the cluster on which {mtc-short} is running.
+* You must add the cluster to the {mtc-short} web console.
+
+.Procedure
+
+. In the left-side navigation pane of the {product-title} web console, click *Projects*.
+. In the list of projects, click your project.
++
+The *Project details* page opens.
+. Click the *DeploymentConfig* name. Note the name of its running pod.
+. Open the YAML tab of the project. Find the PVs and note the names of their corresponding persistent volume claims (PVCs).
+. In the {mtc-short} web console, click *Migration plans*.
+. Click *Add migration plan*.
+. Enter the *Plan name*.
++
+The migration plan name must contain 3 to 63 lower-case alphanumeric characters (`a-z, 0-9`) and must not contain spaces or underscores (`_`).
+
+. In *Migration type*, select *Storage class conversion*.
+. In *Source cluster*, select the desired cluster for storage class conversion.
+. Click *Next*.
++
+The *Namespaces* page opens.
+. Select the required project.
+. Click *Next*.
++
+The *Persistent volumes* page opens. The page displays the PVs in the project, all selected by default.
+. For each PV, select the desired target storage class.
+. Click *Next*.
++
+The wizard validates the new migration plan and shows that it is ready.
+. Click *Close*.
++
+The new plan appears on the *Migration plans* page.
+. To start the conversion, click the three dots menu of the new plan and select *Migrations*.
++
```

Comment:
In the three dots menu, "Migrations" is not a selectable object, It is just a heading (plain text) which is always there and under which the Stage and Cutover are the buttons.

Reply:
Corrected

Reply:
@stillalearner 
Sachin, please approve this PR. Thank you!